### DISCUSSION TITLE
Can't scrub through 'now playing' when in a playlist

### DISCUSSION THREAD

**Discussion by Flamiii** [ppy/osu#15280] [Q&A]
You can scrub through 'now playing' when you're in the main menu and the playlists lounge, but you can't when you're inside of a playlist.

https://user-images.githubusercontent.com/74088315/138618913-63e19363-df36-47cb-a854-7488de478e6d.mp4

**Comment by peppy** [comment]
This is intended behaviour for now. But I guess since you're bringing it up, it's something you'd like to be able to do?

**Reply by Flamiii** [reply]
Personally, I think it would be a nice feature since being able to scrub through a song in a playlist gives me a better idea of what kind of song/map it might be before I actually play it. And overall, I just don't really see how it could hurt.
